inference-engine/thirdparty/clDNN/runtime/ocl/ocl_ext.hpp

```diff
@@ -240,529 +157,551 @@ typedef CL_API_ENTRY cl_mem(CL_API_CALL * PFN_clCreateFromMediaSurfaceINTEL)(
         void* resource, cl_int* errcode_ret);
 #endif
 
-    class SharedSurfLock {
-        cl_command_queue m_queue;
-        std::vector<cl_mem> m_surfaces;
-        cl_int* m_errPtr;
-    public:
-        static PFN_clEnqueueAcquireMediaSurfacesINTEL pfn_acquire;
-        static PFN_clEnqueueReleaseMediaSurfacesINTEL pfn_release;
+class SharedSurfLock {
+    cl_command_queue m_queue;
+    std::vector<cl_mem> m_surfaces;
+    cl_int* m_errPtr;
+public:
+    static PFN_clEnqueueAcquireMediaSurfacesINTEL pfn_acquire;
+    static PFN_clEnqueueReleaseMediaSurfacesINTEL pfn_release;
 
-        static void Init(cl_platform_id platform) {
+    static void Init(cl_platform_id platform) {
 #ifdef WIN32
-            const char* fnameAcq = "clEnqueueAcquireD3D11ObjectsKHR";
-            const char* fnameRel = "clEnqueueReleaseD3D11ObjectsKHR";
+        const char* fnameAcq = "clEnqueueAcquireD3D11ObjectsKHR";
+        const char* fnameRel = "clEnqueueReleaseD3D11ObjectsKHR";
 #else
-            const char* fnameAcq = "clEnqueueAcquireVA_APIMediaSurfacesINTEL";
-            const char* fnameRel = "clEnqueueReleaseVA_APIMediaSurfacesINTEL";
+        const char* fnameAcq = "clEnqueueAcquireVA_APIMediaSurfacesINTEL";
+        const char* fnameRel = "clEnqueueReleaseVA_APIMediaSurfacesINTEL";
 #endif
-            if (!pfn_acquire) {
-                pfn_acquire =
-                    reinterpret_cast<PFN_clEnqueueAcquireMediaSurfacesINTEL>
-                    (clGetExtensionFunctionAddressForPlatform(platform, fnameAcq));
-            }
-            if (!pfn_release) {
-                pfn_release = reinterpret_cast<PFN_clEnqueueReleaseMediaSurfacesINTEL>
-                (clGetExtensionFunctionAddressForPlatform(platform, fnameRel));
-            }
+        if (!pfn_acquire) {
+            pfn_acquire =
+                reinterpret_cast<PFN_clEnqueueAcquireMediaSurfacesINTEL>
+                (clGetExtensionFunctionAddressForPlatform(platform, fnameAcq));
         }
+        if (!pfn_release) {
+            pfn_release = reinterpret_cast<PFN_clEnqueueReleaseMediaSurfacesINTEL>
+            (clGetExtensionFunctionAddressForPlatform(platform, fnameRel));
+        }
+    }
 
-        SharedSurfLock(cl_command_queue queue,
-            std::vector<cl_mem>& surfaces,
-            cl_int * err = NULL)
-            : m_queue(queue), m_surfaces(surfaces), m_errPtr(err) {
+    SharedSurfLock(cl_command_queue queue,
+        std::vector<cl_mem>& surfaces,
+        cl_int * err = NULL)
+        : m_queue(queue), m_surfaces(surfaces), m_errPtr(err) {
 
-            if (pfn_acquire!=NULL && m_surfaces.size()) {
-                cl_int error = pfn_acquire(m_queue,
-                    static_cast<cl_uint>(m_surfaces.size()),
-                    m_surfaces.data(),
-                    0, NULL, NULL);
+        if (pfn_acquire!=NULL && m_surfaces.size()) {
```

Comment:
Please add spaces around "!=".

Reply:
enabled cpplint for runtime and fixed all reported issues